core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-block-editor-sidebar/dot-block-editor-sidebar.component.spec.ts

```diff
@@ -23,6 +22,8 @@ import { CoreWebServiceMock, MockDotMessageService, mockResponseView } from '@do
 import { DotMessagePipeModule } from '@pipes/dot-message/dot-message-pipe.module';
 import { DotBlockEditorSidebarComponent } from '@portlets/dot-edit-page/components/dot-block-editor-sidebar/dot-block-editor-sidebar.component';
 
+const DEFAULT_LANG_ID = 1;
```

Comment:
Was this fixed?

Reply:
It was the fix for the error we had on the editing page.